src/blocks/Contributors/render.php

```
@@ -0,0 +1,392 @@
+<?php
+/**
+ * Render callback for the Contributors block
+ *
+ * @param array    $attributes Block attributes
+ * @param string   $content    Block content
+ * @param WP_Block $block      Block instance
+ * @return string
+ */
+function render_wedocs_contributors_block($attributes, $content, $block) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$content'. (undefined)

(UnusedFormalParameter)